.trivyignore

```diff
@@ -1,19 +1,14 @@
 # telegraf MEDIUM 
 CVE-2019-3826
-CVE-2022-23471
-CVE-2022-41721
-CVE-2023-25153
-CVE-2023-25173
-GHSA-4xgv-j62q-h3rj
-GHSA-hxp2-xqf3-v83h
+CVE-2023-28841
+CVE-2023-28842 
+CVE-2023-25809
 
-#telegraf HIGH
-CVE-2022-41723
+# telegraf HIGH
```

Comment:
what's the plan on fixing these telegraf high vulnerabilities?

Reply:
Working with Eric from Mariner team to figure this out